app/src/main/java/com/martynaskairys/wallpee/ChoosingRandomPicture.java

```diff
@@ -32,15 +44,30 @@ public void onClick(View v) {
                         .load(picture)
                         .placeholder(R.drawable.mok)
                         // .error(R.drawable.ic_error_fallback)
-                         .resize(1250, 1200)
+                        .resize(1250, 1200)
                         // .rotate(90)
                         .into(imageView);
 
 
+
             }
         });
+findViewById(R.id.buttonSetRandomWallpaper).setOnClickListener(new View.OnClickListener() {
+    @Override
+    public void onClick(View v) {
 
+        WallpaperManager wpm = WallpaperManager.getInstance(getApplicationContext());
```

Comment:
this is the same things that RandomlyChangeWallpapers does, right? If so, you can reuse your code rather than duplicate it